07-building-chat-applications/typescript/chat-completions-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "chat-completions-app",
+  "version": "1.0.0",
+  "description": "a simple application how to use the chat completions with Azure OpenAI in TypeScript",
+  "main": "main.js",
+  "scripts": {
+    "build": "rimraf build && tsc",
+    "preserve": "npm run build",
+    "start": "npm run build && cross-env NODE_ENV=development concurrently \"nodemon\"",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "typescript",
+    "azure",
+    "openai"
+  ],
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/node": "^20.10.8",
+    "concurrently": "^8.2.2",
+    "nodemon": "^3.0.2",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.3.3"
+  },
+  "dependencies": {
+    "@azure/openai": "^1.0.0-beta.10",
+    "cross-env": "^7.0.3",
```

Comment:
```suggestion
```

Reply:
Thanks a lot for this, @manekinekko. I included all your suggestions in the PR :)